build-config/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -207,6 +208,11 @@
       <property name="message" value="Closing parenthesis should be on the same line as the last parameter"/>
       <property name="illegalPattern" value="true"/>
     </module>
+    <module name="Regexp">
+      <property name="format" value="@PropertyDefinition(?!(.*validate = |.*get = \&quot;optional&quot;))[\S ]*(\n|\r\n)(?![\S ]* (byte|char|short|int|long|float|double|boolean) [\S ]*$)"/>
+      <property name="message" value="Properties should either be declared 'validate = &quot;notNull&quot;' or 'get = &quot;optional&quot;'"/>
```

Comment:
So does this match any property definition with a `validate = ` section?
Looking and it seems there's a lot of code where this is going to need to be fixed, spread across all projects. Are you sure you want to do that update?

Reply:
I don't think we should shy away from fixing code that doesn't conform to the standards. I'm more concerned with getting conformation that the rule change appropriately captures those standards.